axlearn/open_api/metrics/tool_use_execution.py

```diff
@@ -312,13 +472,25 @@ def get_tool_calls_from_message(message: dict[str, Any]) -> list[dict[str, Any]]
                 and len(target.tool_calls) == len(pred.tool_calls)
             ):
                 pred_tool_calls = get_tool_calls_from_message(pred.model_dump())
-                target_tool_calls = get_tool_calls_from_message(target.model_dump())
                 match_rules = response.get("target_message_match_rules", None)
                 matched = _compare_tool_calls(
                     pred_tool_calls=pred_tool_calls,
                     target_tool_calls=target_tool_calls,
                     match_rules=match_rules,
                 )
+
+            if target.tool_calls is not None and pred.tool_calls is not None:
+                # Detailed matching
+                # if target.tool_calls is not None and pred.tool_calls is not None:
```

Comment:
Remove the commented code and clarify the comment on L483 with full sentences.

Reply:
Done